server/apolloassistant/verification.py

```diff
@@ -0,0 +1,34 @@
+from django.conf import settings
+
+from Crypto.PublicKey import RSA
+from Crypto.Signature import PKCS1_PSS
+from Crypto.Hash import SHA
+
+
+def generate_RSA_pair(size=2048):
+    settings.PRIVATE_KEY=RSA.generate(size)
+    settings.PUBLIC_KEY=settings.PRIVATE_KEY.publickey()
+    with open(f"{settings.REGISTER_KEYS_PATH}/private_key.pem", "w") as pem_file:
+        pem_file.write(settings.PRIVATE_KEY.exportKey('PEM'))
+    with open(f"{settings.REGISTER_KEYS_PATH}/public_key.pem", "w") as pem_file:
+        pem_file.write(settings.PUBLIC_KEY.exportKey('PEM'))
```

Comment:
What are does? How their are different that the two keys in lines 9-10. Why are we loading them on demand instead of loading them once when apps starts from env?